### PR TITLE
Adds support of Pistol Grip to start and stop barcode scan

### PR DESCRIPTION
**Pull request by FlorentYvinec**

### CODE REVIEW

File: app/src/main/java/fr/coppernic/scansample/MainActivity.java [hunk 1]
```diff
@@ -265,4 +295,31 @@ private void displayServiceStatus(boolean start) {
...
+            }
+            scanIsStopped = false;
+            return false;
+        }
+        return super.onKeyDown(keyCode, keyEvent);
+    }
+    @Override
+    public boolean onKeyUp(int keyCode, KeyEvent keyEvent) {
+        Timber.d("onKeyUp()" + keyCode);
+        // Change
```
Comment (mreynier) [inline app/src/main/java/fr/coppernic/scansample/MainActivity.java]: why change? A comment saying that it is for Access trigger gun?

File: app/src/main/java/fr/coppernic/scansample/MainActivity.java [hunk 2]
```diff
@@ -32,6 +33,9 @@ public class MainActivity extends AppCompatActivity {
     public static final String ACTION_SCAN_SUCCESS = "fr.coppernic.intent.scansuccess";
     public static final String ACTION_SCAN_ERROR = "fr.coppernic.intent.scanfailed";
     public static final String INTENT_ACTION_SCAN = "fr.coppernic.intent.action.SCAN";
+
```
Comment (mreynier) [inline app/src/main/java/fr/coppernic/scansample/MainActivity.java]: pkoi des ligne vide ici

**Review by mreynier — APPROVED** [review]

File: app/src/main/java/fr/coppernic/scansample/MainActivity.java (hunk 2 above)
Comment (FlorentYvinec) [reply app/src/main/java/fr/coppernic/scansample/MainActivity.java]: Merci !

File: app/src/main/java/fr/coppernic/scansample/MainActivity.java (hunk 1 above)
Comment (FlorentYvinec) [reply app/src/main/java/fr/coppernic/scansample/MainActivity.java]: Merci !

**Pull request merged**